string id_frame
uint32 left_speed
uint32 right_speed
uint8 left_alarm
uint8 right_alarm    
uint8 remote_controlled
